src/select/Select.ts

```diff
@@ -0,0 +1,506 @@
+import { html, css, nothing } from 'lit';
+import { customElement, property, query, state } from 'lit/decorators.js';
+import { live } from 'lit/directives/live.js';
+import { until } from 'lit/directives/until.js';
+import { OmniFormElement } from '../core/OmniFormElement.js';
+import { RenderFunction } from '../render-element/RenderElement.js';
+import '../icons/Loading.icon.js';
+import '../render-element/RenderElement.js';
+import '../icons/ChevronDown.icon.js';
+import '../icons/More.icon.js';
+
+/**
+ * A select input control.
+ *
+ * ```js
+ *
+ * import '@capitec/omni-components/select';
+ * ```
+ * @example
+ *
+ * ```html
+ * <omni-select
+ *   label="Enter a value"
+ *   value="Hello World"
+ *   data="{'id': 12345, 'name': 'Test'}"
+ *   hint="Required"
+ *   error="Field level error message"
+ *   items={'item1','item2','item3','item4'}
+ *   display-field="label"
+ *   id-field="id"
+ *   renderItem=() => {}
```

Comment:
Jsdoc examples should always be pure html versions. Functions are not valid attributes, don't document `renderItem` as part of the example